inc/functions/files.php

```diff
@@ -52,9 +52,10 @@ function rocket_generate_advanced_cache_file( $advanced_cache = null ) {
  *
  * @since 2.0
  *
+ * @param bool $cache_mobile Mobile Cache config value.
  * @return array Names of all config files & The content that will be printed
  */
-function get_rocket_config_file() { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
+function get_rocket_config_file( $cache_mobile = true ) { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
```

Comment:
why is this needed?

Reply:
To disable the mobile cache options when the `pre_get_rocket_option` is fired